generators/app/templates/_generator.yml

```
@@ -0,0 +1,21 @@
+name: Generator
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  generator-jhipster-<%= moduleName %>:
+    name: npm-test
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2.1.2
```

v2.1.2 is beta, in this case we should use v2 tag instead.
```suggestion
      - uses: actions/setup-node@v2
```

---

great! thanks